pkg/k8sutil/k8sutil.go

```diff
@@ -183,3 +184,44 @@ func isKubeMetaKind(kind string) bool {
 func isRunModeLocal() bool {
 	return os.Getenv(ForceRunModeEnv) == string(LocalRunMode)
 }
+
+// SupportsOwnerReference func checks whether a given dependent supports owner references, based in the owner.
+func SupportsOwnerReference(restMapper meta.RESTMapper, owner runtime.Object, dependent runtime.Object) (bool, error) {
```

Comment:
Nit:
```suggestion
func SupportsOwnerReference(restMapper meta.RESTMapper, owner, dependent runtime.Object) (bool, error) {
```

Reply:
Should not better use the default nomenclature owner/child?

Reply:
@camilamacedo86 Is your suggestion same as what Eric has suggested above?

Reply:
My suggestion is to replace the name `dependent` for `child` which is the nomenclature used for the watch annotations implemented for Ansible. I mean, adopted the same nomenclature as possible.  See: https://github.com/operator-framework/operator-sdk/blob/master/pkg/ansible/proxy/proxy.go#L251

Reply:
nit, just wanted to note that owner/dependent is consistent with the broader Kubernetes documentation https://kubernetes.io/docs/concepts/workloads/controllers/garbage-collection/#owners-and-dependents